src/Services/FHIR/FhirConditionService.php

```diff
@@ -66,31 +66,32 @@ public function parseOpenEMRRecord($dataRecord = array(), $encode = false)
 
         $clinicalStatus = "inactive";
         $clinicalSysytem = "http://terminology.hl7.org/CodeSystem/condition-clinical";
-        if (
-            (!isset($data['enddate']) && isset($data['begdate']))
-            || isset($data['enddate']) && strtotime($data['enddate']) >= strtotime("now")
+        if ((!isset($dataRecord['enddate']) && isset($dataRecord['begdate']))
+            || isset($dataRecord['enddate']) && strtotime($dataRecord['enddate']) >= strtotime("now")
```

Comment:
```
FILE: /home/travis/build/openemr/openemr/src/Services/FHIR/FhirConditionService.php
------------------------------------------------------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------------
 69 | ERROR | [x] The first expression of a multi-line control structure must be on the line after the opening
    |       |     parenthesis
------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
------------------------------------------------------------------------------------------------------------------------
```
does this make it work:
```
        if (
            (!isset($dataRecord['enddate']) && isset($dataRecord['begdate']))
            || isset($dataRecord['enddate']) && strtotime($dataRecord['enddate']) >= strtotime("now")
        ) {
```

Reply:
if not, then i can "autofix" this after we bring the pull request in

Reply:
I was using this https://www.webcodesniffer.net/ and it gave me error for the previous one.  I will install the tools mentioned